ros_buildfarm/templates/release/sourcedeb_job.xml.em

```diff
@@ -174,7 +177,7 @@
 ))@
 @(SNIPPET(
     'build-wrapper_ssh-agent',
-    credential_id=credential_id,
+    credential_id=git_ssh_credential_id,
```

Comment:
Removing the `credential_id` here breaks the sourcedeb jobs. Addressed in 7112d447c0d076e076fc78fc6237f734240031a4.